code/planning/src/planning/help_functions.py

```diff
@@ -0,0 +1,124 @@
+from typing import Tuple
+from math import floor, asin, sqrt, cos, sin, pi
+import numpy as np
+
+
```

Comment:
for all theses different functions it might be nice to have testing units that validate their functionality or some kind of reference where they are from

Reply:
Yes i agree with you.